### PR TITLE
[214] Add FAQ section into Readme

### PR DESCRIPTION
**Pull request by nvminhtue**
https://github.com/nimblehq/elixir-templates/issues/214

## What happened

The new beginner might get struggling with the first initialize project using Elixir-Template along with Nimble's laptop-script

## Insight

Adding the FAQ section and the very first question to cover errors might get when initializing the project without installing the Phoenix application generator.

## Proof Of Work

Screenshot of the new coming FAQ section either the 2 first Q&A.

image

### CODE REVIEW

**Comment by Nihisil** [comment]
Maybe while we are here, it is worth adding second section to FAQ as well: https://github.com/nimblehq/elixir-templates/issues/202

File: README.md [hunk 1]
```diff
@@ -96,3 +96,17 @@ See [our other projects][community] or [hire our team][hire] to help build your
 
 [community]: https://github.com/nimblehq
 [hire]: https://nimblehq.co
+
+## FAQ
+
+### 1. I'm having Mix, Elixir, and Erlang/OTP but can not initialize project by the raise of the error `(Mix) The task "phx.new" could not be found`
```
Comment (andyduong1920) [inline README.md:102]: How about just saying 

### 1. Getting `(Mix) The task "phx.new" could not be found` error.

The Phoenix application generator is missing, install it by

```bash
# Latest Phoenix version
mix archive.install hex phx_new

# Specific Phoenix version
mix archive.install hex phx_new #{specific-phoenix-version}
```

File: README.md [hunk 2]
```diff
@@ -96,3 +96,17 @@ See [our other projects][community] or [hire our team][hire] to help build your
 
 [community]: https://github.com/nimblehq
 [hire]: https://nimblehq.co
+
+## FAQ
```
Comment (andyduong1920) [inline README.md:100]: should we move this FAQ to above the `## Contributing`,  or just right after the `## Usage` part 🤔

File: README.md (hunk 1 above)
Comment (nvminhtue) [reply README.md:102]: Sure I will adjust that one.

**Comment by nvminhtue** [comment]
@Nihisil Sure, I will add your mentioned prerequisite, stay tuned.

File: README.md [hunk 3]
```diff
@@ -79,6 +79,19 @@ Once the release branch is merged into the `master` branch, Github Action automa
...
+
+### 1. Getting `(Mix) The task "phx.new" could not be found` error
+
+The Phoenix application generator is missing. By solving this problem, you need to run
+``` bash
+mix archive.install hex phx_new
+```
+or
+```bash
```
Comment (andyduong1920) [inline README.md:91]: @nvminhtue a small comment, can we add the `line break` between the code block 🙏

```

``` bash
mix archive.install hex phx_new
```

or

``` bash
xxx
```
```

**Review by andyduong1920 — APPROVED** [review]

**Comment by nvminhtue** [comment]
@andyduong1920 @Nihisil 
As mentioned, I've resolved the comment and also added the issue raised from #202.
Please take a look and feel free to drop some comments 🙏

**Review by andyduong1920 — APPROVED** [review]

**Comment by andyduong1920** [comment]
> @andyduong1920 @Nihisil As mentioned, I've resolved the comment and also added the issue raised from #202. Please take a look and feel free to drop some comments 🙏

LGTM @nvminhtue, @Nihisil can you have a look on this PR again 🙏

**Review by Nihisil — APPROVED** [review]

**Comment by Nihisil** [comment]
Looks good! Thank you

**Comment by andyduong1920** [comment]
> Looks good! Thank you

Actually, we could add that content into the `generated project` README and Wiki as well as the user will face that issue from the `generated project`, I can open a PR for that after this PR :)

**Pull request merged**